packages/truffle/test/scenarios/compile/compile.js

```diff
@@ -378,12 +385,15 @@ describe("Compilation with db enabled", async () => {
     `;
 
     // connect to DB
-    const db = connect(config);
+    const db = connect();
     const results = await db.execute(GetAllContracts, {});
 
     // number of contracts matches number of contracts in the project directory
     // (plus one library in this one)
-    assert(results.data.contracts.length === 4);
+    const names = ["Contract", "InnerLibrary", "Migrations", "RelativeImport"];
+    for (const name of names) {
+      assert(results.data.contracts.some(contract => contract.name === name));
+    }
```

Comment:
Ooh a free test improvement :D

Reply:
Well this also needed to be changed because it looked like other contracts were now showing up for this query since everything is kept here now.